tests/data/fields/label_field_test.py

```diff
@@ -32,3 +32,7 @@ def test_label_field_raises_with_non_integer_labels_and_no_indexing(self):
     def test_label_field_raises_with_incorrect_label_type(self):
         with pytest.raises(ConfigurationError):
             _ = LabelField([], skip_indexing=False)
+
+    def test_label_field_empty_field_works(self):
+        label = LabelField("test")
+        label.empty_field()
```

Comment:
Better if you actually check something here.  E.g., it's probably good to put the `-1` for padded labels in a test, so that if someone wants to change it they break a test.  That encourages backwards-compatibility, and makes someone think harder before making an incompatible change.